agent/installer/steps.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package installer
+
+// contains the installation and uninstallation steps for the supported os and k8s
+var (
+	DoUbuntu20_4K8s1_22 = `
+set -euo pipefail
+
+BUNDLE_PATH=${BUNDLE_PATH:-"/var/lib/byoh/bundles"}
+
+## disable swap
+swapoff -a && sed -ri '/\sswap\s/s/^#?/#/' /etc/fstab
```

Comment:
I was assuming these scripts will be generated by the agent installer and then assigned to `DoUbuntu20_4K8s1_22 ` variable.

Aren't we duplicating these scripts, one in the agent installer and the other in these variables?

Reply:
Agreed, for some time these script will be duplicated.

we are planing to update the byo agent to install the bundle two ways:
1. agent will install the way its currently doing
2. new `K8sInstallationConfig` controller will generate the installation script and agent will execute it

These variables will be used by new `K8sInstallationConfig` controller to create the secret.